runner/src/main/kotlin/pl/edu/prz/kod/runner/adapters/http/CustomUndertow.kt

```diff
@@ -0,0 +1,69 @@
+package pl.edu.prz.kod.runner.adapters.http
+
+import io.undertow.Undertow
+import io.undertow.UndertowOptions.ENABLE_HTTP2
+import io.undertow.server.HttpServerExchange
+import io.undertow.server.handlers.BlockingHandler
+import io.undertow.server.handlers.GracefulShutdownHandler
+import org.http4k.core.HttpHandler
+import org.http4k.server.ServerConfig.StopMode
+import org.http4k.server.Http4kServer
+import org.http4k.server.Http4kUndertowHttpHandler
+import org.http4k.server.ServerConfig
+import java.net.InetSocketAddress
+
+class CustomUndertow(
+    val port: Int = 8000,
+    val enableHttp2: Boolean,
```

Comment:
Is this value needed if it's always going to be `false` anyways?

Reply:
Dunno, I'll remove it.